### PR TITLE
Webinar slides and documentation refactor

### PR DESCRIPTION
**Pull request by terrywbrady**
- Create GitPitch slides for Duraspace Webinar
- Create tutorials for repository managers
- Replace complicated tutorial pages with simplified/refactored pages
- Document "Common Tasks" for a user
- Create compose file to run a 2nd DSpace instance on a separate port

### CODE REVIEW

File: PITCHME.css [hunk 1]
```diff
@@ -1,6 +1,4 @@
-code {
-  background-color: #EEEEEE;
-  color: red;
-  display: inline-block;
+.reveal pre {
+  width:96%;
```
Comment (PTrottier) [inline PITCHME.css]:
```suggestion
  width: 96%;
```
Just to keep things consistent

File: documentation/setup.LinuxDesktop.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+sudo yum install git
+```
+
+#### Debian, Ubuntu
+
+```shell
+sudo apt-get install git
+```
+
+#### Debian
```
Comment (PTrottier) [inline documentation/setup.LinuxDesktop.md]:
```suggestion
#### Fedora
```
From my experience in Linux, Fedora is the distribution that uses `dnf`, whereas Debian uses `apt` as shown above

File: PITCHME.md [hunk 3]
```diff
@@ -231,78 +245,130 @@ DSpace 6 Overrides
...
 +++
 
+### What ideas do you have?
+
+---
+
+### Reference Links
+
+- [DSpace Docker Tutorial Overview](https://dspace-labs.github.io/DSpace-Docker-Images/)
+- Docker tutorial
```
Comment (PTrottier) [inline PITCHME.md]: Should these links be populated as well?
```suggestion
- [Docker tutorial](https://dspace-labs.github.io/DSpace-Docker-Images/documentation/tutorialSetup.html)
```

File: PITCHME.md [hunk 4]
```diff
@@ -231,78 +245,130 @@ DSpace 6 Overrides
...
+- Docker compose tutorial
```
Comment (PTrottier) [inline PITCHME.md]: See previous
```suggestion
- [Docker compose tutorial](https://dspace-labs.github.io/DSpace-Docker-Images/docker-compose-files/dspace-compose/)
```

File: PITCHME.md [hunk 5]
```diff
@@ -231,78 +245,130 @@ DSpace 6 Overrides
...
+- Join the **#dspace-docker** Slack Channel
```
Comment (PTrottier) [inline PITCHME.md]: See previous
```suggestion
- Join the [**#dspace-docker**](https://dspace-org.slack.com/messages/C9YD42PV3/) Slack Channel
```

**Comment by terrywbrady** [comment]
@PTrottier , I appreciate the review.  Thank you.

I started the slideshow from the beginning (after a page reload) and the slides seem to advance as I expected.

Since slides advance in 2 directions in GitPitch, it can be confusing if you go backwards after advancing down and to the right.

**Comment by terrywbrady** [comment]
@pnbecker , I merged this into master but I am planning to keep the branch open until after the webinar.

Once the webinar is done, we can create a release named "webinar" and delete the branch.

**Pull request merged**